org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -91,6 +91,11 @@ public String handleRequest(Map<String, String> lambdaInput, Context context) {
             dashboardHostname,
             useNeighborhood);
 
+    // manually set font configuration file since there is no font configuration on a lambda.
+    java.util.Properties props = System.getProperties();
+    // /opt is the folder all layer files go into.
+    props.put("sun.awt.fontconfig", "/opt/fontconfig.properties");
```

Comment:
(possibly out of scope) - we also set the connection ID on a global properties object that we maintain (see `setConnectionId` above). Should that move to this model as well?

Reply:
good question. I'm not sure how the two relate. This is overwriting a property on the jvm to tell it to look in that location for the fontconfig file, instead of the default (which on the lambda does not exist). How do we use the properties object for the connection id?

Reply:
The properties object used for the connection id essentially sets a global variable that needs to be available across the various packages.

Reply:
makes sense, I think these might be separate cases but we can take a look as a follow-up as to whether we can put the connection id in the system properties